cspatients/tests.py

```diff
@@ -320,29 +353,34 @@ def test_get_all_active_patient_entries(self):
         save_model(self.patient_one_data)
         save_model(self.patient_two_data)
         entry3, _ = save_model(self.patient_three_data)
+        entry4, _ = save_model(self.patient_four_data)
 
         entry3.completion_time = timezone.now()
         entry3.save()
 
+        entry4.decision_time = timezone.now() + timezone.timedelta(hours=1)
+        entry4.save()
+
         patient_entries = get_all_active_patient_entries()
 
-        # Check that it returns two objects
-        self.assertEqual(len(patient_entries), 3)
+        # Check that it returns 4 objects
+        self.assertEqual(len(patient_entries), 4)
 
         # Check that the results are sorted by the urgency
-        self.assertEqual(patient_entries[0].patient.name, "Jane Doe")
-        self.assertEqual(patient_entries[1].patient.name, "Mary Mary")
-        self.assertEqual(patient_entries[2].patient.name, "John")
+        self.assertEqual(patient_entries[0].patient.name, "Urgency Immediate NEW")
+        self.assertEqual(patient_entries[1].patient.name, "Urgency Immediate")
+        self.assertEqual(patient_entries[2].patient.name, "Urgency COLD")
+        self.assertEqual(patient_entries[3].patient.name, "John Completed")
 
     def test_get_all_active_patient_entries_filter(self):
         save_model(self.patient_one_data)
         save_model(self.patient_two_data)
         save_model(self.patient_three_data)
 
-        patient_entries = get_all_active_patient_entries("jane", "4")
+        patient_entries = get_all_active_patient_entries("COLD", "4")
 
         self.assertEqual(len(patient_entries), 1)
-        self.assertEqual(patient_entries[0].patient.name, "Jane Doe")
+        self.assertEqual(patient_entries[0].patient.name, "Urgency COLD")
```

Comment:
Just want to check why we're using status where you would normally have name?

Reply:
Yes just gave them a more descriptive name to make sure the sorting is working as expected.